content/blog/youtrack-github.md

```diff
@@ -0,0 +1,36 @@
+---
+author: "Alex Hill"
+date: 2019-04-17
+title: Integrating YouTrack and GitHub workflows
+best: false
+---
+
+Like most technical teams we use an issue tracker to manage our work. Our tool of choice for
+this is [YouTrack](https://www.jetbrains.com/youtrack/). We also make extensive use of GitHub and to synchronise our 
+workflow across the two platforms we name our git branches with YouTrack ticket ids, so every pull request 
+corresponds to a ticket. This naming convention makes it easier to track work in progress across the two platforms, 
+plus our deploy tool can use git to look at which branches have been merged in and then auto-update the 
```

Comment:
"our deploy tool" is likely to be unclear to some people - perhaps something like:

> when we deploy our software, we have a tool that uses git to look at...